habitat-aspnet-sample/plan.sh

```diff
@@ -0,0 +1,49 @@
+pkg_name=habitat-aspnet-sample
+pkg_origin=core
+pkg_version=0.1.9
+pkg_source=https://github.com/mwrock/${pkg_name}/archive/v${pkg_version}.tar.gz
+pkg_shasum=50ffcbe1c0b49acb84365c2157445047b360345c7361e67827093f962852de4d
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=('MIT')
```

Comment:
Missing a `pkg_upstream_url`. Either use your GitHub repo's web address (if you're going with that) or docs about the `dotnet new` thing maybe or just the website for dotnet-core.